zuul.d/edpm_periodic.yaml

```diff
@@ -3,6 +3,10 @@
     name: periodic-podified-edpm-deployment-master-ocp-crc-1cs9
     parent: cifmw-crc-podified-edpm-deployment
     vars: &edpm_vars
+      cifmw_repo_setup_branch: master
+      cifmw_repo_setup_promotion: podified-ci-testing
+      cifmw_tempest_image: quay.rdoproject.org/podified-main-centos9/openstack-tempest
+      cifmw_tempest_image_tag: "{{ cifmw_repo_setup_dlrn_hash_tag | default(cifmw_repo_setup_full_hash) }}"
```

Comment:
we can move these vars to https://github.com/openstack-k8s-operators/ci-framework/blob/main/scenarios/centos-9/edpm_periodic.yml instead of defining here.

Reply:
This will work for tempest but not dlrn reporting as that is done in post and scenarios are not passed to post.
Testproject run skips dlrn reporting as its false by default